addons/block_code/blocks/input/is_input_actioned.gd

```diff
@@ -0,0 +1,35 @@
+@tool
+extends BlockExtension
+
+const OptionData = preload("res://addons/block_code/code_generation/option_data.gd")
```

Comment:
**const OptionData** has been moved to here.  Should the [original one](https://github.com/endlessm/godot-block-coding/pull/222/files#diff-ec1a3a88731ab0f21c32be474930ad0bceb7078d57c246d16a3c50ebd9dffadeL4) in "addons/block_code/code_generation/blocks_catalog.gd" be dropped?

Reply:
:+1: